docs/processors/standalone/wasm.mdx

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 2
+---
+
+# How it works under the hood (Wasm)
```

Comment:
I'm wondering if we should simply could call it "How it works" and then explain in the document why using Wasm in the first place.

I think that could still valuable to explain the versatility this brings.

Reply:
I was expecting a comment about the title, I'm not too happy with it myself. `How it works` sounds better, the `under the hood` part is kinda redundant. I'm wondering if there's a way to include `Wasm` in the title without making it too long? 🤔 I feel like with that in the title you already get an idea that standalone processors are Wasm modules, without even opening the page.

I'll add an explanation on top why Wasm is a good choice 👍

Reply:
Done!